ui/Dockerfile

```diff
@@ -48,26 +50,26 @@ FROM $BASE_IMAGE_REGISTRY/chainguard/wolfi-base:latest AS final
 ENV LANG=C.UTF-8
 ENV LC_ALL=C.UTF-8
 RUN apk update \
-    && apk add curl bash openssl ca-certificates nginx supervisor \
+    && apk add curl bash openssl unzip ca-certificates nginx supervisor \
```

Comment:
why do we need unzip here?

also, nginx and supervisor are being installed on line 8 -- should they be removed from here?